### DISCUSSION TITLE
Manually run the cross-seed

### DISCUSSION THREAD

**Discussion by ajhorvath20** [cross-seed/cross-seed#71] [Q&A] [answered]
Hi, currently I can run it in `search` or `daemon` mode but I am not sure how can I run it manually. Is it possible to do it from browser or inside the container when in `daemon` mode?

Thanks

**Comment by mmgoodnow** [comment, marked as answer]
Currently no. Your best option is to spin up another docker container with a `docker run` command. 

The way I plan to solve for this use-case is to 
- eventually support reading tracker RSS feeds and matching those with existing torrents in your client; this would sort of be like autodl but for cross-seeding (#13)
- allow a way to have search mode run weekly or so (probably based on a cron schedule string), while also running daemon mode. (no issue for this, feel free to open one)
